source/extensions/filters/common/ext_authz/ext_authz_http_impl.cc

```diff
@@ -279,7 +280,8 @@ void RawHttpClientImpl::check(RequestCallbacks& callbacks,
     auto options = Http::AsyncClient::RequestOptions()
                        .setTimeout(config_->timeout())
                        .setParentSpan(parent_span)
-                       .setChildSpanName(config_->tracingName());
+                       .setChildSpanName(config_->tracingName())
+                       .setSampled(absl::nullopt);
```

Comment:
Could you add a comment here on why this is helping on keeping the parent sampling status?

Reply:
Sure thing, done.